plugins/mcreator-core/variables/dyecolor.json

```diff
@@ -0,0 +1,4 @@
+{

```

Comment:
Variable will need to define set and get inside variable yaml for both generators to work properly